extensions/2.0/Vendor/GRIFFEL_bim_data/schema/GRIFFEL_bim_data.schema.json

```diff
@@ -0,0 +1,105 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "GRIFFEL_bim_data Serializable Object",
+    "type": "object",
+    "description": "Contains same properties as embedded version plus properties to node map. Used only when data is saved in a separate file.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "nodeProperties": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "description": "Node properties and type.",
+                "properties": {
+                    "node": {
+                        "allOf": [ { "$ref": "glTFid.schema.json" } ],
+                        "description": "Index of the node in glTF file."
+                    },
+                    "properties": {
+                        "type": "array",
+                        "item": {
+                            "allOf": [ { "$ref": "glTFid.schema.json" } ],
+                            "description": "Index of a property in the root level collection."
+                        },
+                        "description": "Collection of indices properties of the node.",
+                        "minItems": 1
+                    },
+                    "type":{
+                        "allOf": [ { "$ref": "glTFid.schema.json" } ],
+                        "description": "Index of the type of the node."
+                    }
+                },
+                "required": [ "properties" ]
+            },
+            "description": "Node type and instance properties map. Used only when data is saved in a separate file."
+        },
+        "propertyValues": {
+            "type": "array",
+            "items": {
+                "type": "string",
+                "description": "Property value.",
+                "minLength": 1
+            },
+            "minItems": 1,
+            "description": "Collection of unique property values."
+        },
+        "propertyNames": {
+            "type": "array",
```

Comment:
Since storing the same property name more than once contradicts the intended usage of this array, consider adding `"uniqueItems": true` here.

Reply:
Done.